oak-run/src/main/js/oak-mongo.js

```diff
@@ -452,6 +452,70 @@ var oak = (function(global){
         }
     };
 
+    /**
+     * Removes all unmerged branches on the document with the given path and
+     * clusterId. This method will only remove unmerged branches when the
+     * clusterId is inactive.
```

Comment:
Should we add a comment a la
"This corresponds to DocumentNodeStore.cleanOrphanedBranches (which is part of a startup and normal background update)"